### PR TITLE
Block permission front-end

### PR DESCRIPTION
**Pull request by alxnddr**
# Description

[Product doc](https://www.notion.so/metabase/Add-a-Block-permission-to-data-access-d7acb416972e42899018833dc827df00)

The main purpose of adding the block permission is to allow admins to completely block access to questions that use a specific data source. Currently, if users have "No access" permission to a data source, they can still view questions that use the data source. To make a clearer distinction between the new "Block" and the old "No access" option the last one was renamed to "No self-service".

Changes:
- adds new database level "Block" permission
- rename existing permissions according to product doc
- show "Native query editing" permission column for schemas and tables as well (disabled only)
- adds data permissions help reference

Out of the scope:
- Hide questions/cards that use a blocked data source — it requires BE changes mostly

### Screenshot
Screen Shot 2021-08-31 at 00 10 42

### CODE REVIEW

**Comment by alxnddr** [comment]
@metabase-bot run visual tests

**Comment by alxnddr** [comment]
@metabase-bot run visual tests

**Review by daltojohnso — APPROVED** [review]
looks good!

File: frontend/src/metabase/admin/permissions/components/DataPermissionsHelp/DataPermissionsHelp.jsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  DataPermissionsHelpContent,
+  DataPermissionsHelpLink,
+  DataPermissionsHelpLinkIcon,
+} from "./DataPermissionsHelp.styled";
+
+export const DataPermissionsHelp = () => (
+  <DataPermissionsHelpRoot>
+    <DataPermissionsHelpContent>
+      <h2>{t`About data permissions`}</h2>
+      <p>{t`Each of your user groups can has a level of access for each of your databases on the tables they contain.`}</p>
```
Comment (nemanjaglumac) [inline frontend/src/metabase/admin/permissions/components/DataPermissionsHelp/DataPermissionsHelp.jsx:20]:
```suggestion
      <p>{t`Each of your user groups can have a level of access for each of your databases on the tables they contain.`}</p>
```

Comment (alxnddr) [reply frontend/src/metabase/admin/permissions/components/DataPermissionsHelp/DataPermissionsHelp.jsx]: Thanks for catching 👍

**Comment by alxnddr** [comment]
@metabase-bot run visual tests

**Pull request merged**